students/PeterB/Session2/series2.py

```diff
@@ -0,0 +1,31 @@
+# fibonacci
+
+a, b = 0,1
```

Comment:
For the Fibonacci and Lucas series, you would want to create a function similar to how sum_series does it.